contacts.html

```diff
@@ -31,7 +31,7 @@
 				<div class="input-field">
 					<span class="field-text" id="searchFieldText">SEARCH:</span>
 					<input type="text" id="searchText" placeholder="Search..."/>
-					<button type="button" id="searchContactsButton" class="button" onclick="searchContacts();">Search Contacts</button>
+					<button type="submit" id="searchContactsButton" class="button" onclick="searchContacts();">Search Contacts</button>
 				</div>
```

Comment:
Not sure why the type was changed, but kept current type that's on the server just incase.